Documentation/guides/building-apps/build-properties.md

```diff
@@ -1674,6 +1686,13 @@ as support for `$(AotAssemblies)` will be removed in a future release.
 
 Extra options to pass to `aprofutil`.
 
+## BeforeBuildAndroidAssetPacks
+
+MSBuild Targets listed in this
+property will run directly before `_BuildAssetPacks`.
```

Comment:
This is leaking implementation details; we shouldn't mention `_BuildAssetPacks`.

Additionally, what should the integration point be?  Overriding an MSBuild property, or using BeforeTargets/AfterTargets?

Would it be "better" to have a publicly documented "BuildAndroidAssetPacks" ("CreateAndroidAssetPacks"?) target, which can then be used with BeforeTargets/AfterTargets?

(I just find property overrides to frequently be cumbersome to use, and prone to breakage, e.g. if you override the propeprty and don't include the previous property value in the new property value, you might break things.)

Reply:
We can't use `UpdateAndroidAssets` as it happens to early in the build process. 
I can introduce a new public target for the assetpack stuff though.